test/unit/services/aws/sts/assume_role_web_identity_service_test.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class Aws::Sts::AssumeRoleWebIdentityServiceTest < ActiveSupport::TestCase
+  setup do
+    File.stubs(:exist?).returns(true)
+    assume_role_response.class.any_instance.stubs(:expiration).returns(1.minute.from_now)
+    assume_role_response.stubs(:credentials).returns(sts_credentials)
+  end
+
+  test '#config sets params' do
+    assert(
+      assume_role_web_identity_service_instance.config(sts_auth_params).as_json['params'].keys.sort,
```

Comment:
I agree it's more readable. I usually avoid using `send` so I don't call private methods directly. Now, since this is a test, I'm OK with the change. Updated in c003f85 :v: